test/sandbox/sandbox.go

```diff
@@ -77,7 +77,7 @@ type FileEntry struct {
 func New(t *testing.T) S {
 	t.Helper()
 
-	basedir := t.TempDir()
+	basedir := test.CanonPath(t, t.TempDir())
```

Comment:
maybe we could rename basedir to rootdir on the sandbox, would be more aligned with the new design.